.cloudbuild/vep-docker.cloudbuild.yaml

```diff
@@ -1,11 +1,11 @@
 # Run locally with:
 #
-# gcloud builds submit --quiet --substitutions='_VEP_VERSION=110' --config .cloudbuild/vep-docker.cloudbuild.yaml v03_pipeline/deploy
+# gcloud builds submit --quiet --substitutions='_REFERENCE_GENOME=GRCh38' --config .cloudbuild/vep-docker.cloudbuild.yaml v03_pipeline/deploy
```

Comment:
I've left automated builds to another ticket to avoid scope creep:

https://github.com/broadinstitute/seqr-loading-pipelines/issues/848